src/components/swap/ConfirmSwapModal.tsx

```diff
@@ -303,6 +310,9 @@ export default function ConfirmSwapModal({
     })
 
   const swapStatus = useSwapTransactionStatus(swapResult)
+  const order = useOrder(swapResult?.type === TradeFillType.UniswapX ? swapResult.response.orderHash : '')
```

Comment:
nit: rename `uniswapXOrder` to be more explicit

Reply:
Renamed